cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/buildah/buildah-task.yaml

```diff
@@ -108,13 +108,12 @@ spec:
           exit 1
         fi
       fi
-
+      IMAGE=$(params.IMAGE)
       buildah --storage-driver=$(params.STORAGE_DRIVER) push \
         $(params.PUSH_EXTRA_ARGS) --tls-verify=$(params.TLSVERIFY) \
-        --digestfile /tmp/image-digest $(params.IMAGE) \
-        docker://$(params.IMAGE)
+        --digestfile /tmp/image-digest $IMAGE docker://$IMAGE
```

Comment:
@jordigilh thanks for the PR, I see we use `$(params.IMAGE)` on line number 90. can you please update that entry too?

Reply:
Done